services/api/mw2pdf/INSTALL.md

```diff
@@ -1,8 +1,12 @@
 # Installing
 
 This script requires node.js to be installed.
-In order to run the script, you will first need to build the dependencies using `npm install`
+In order to run the script, you will first need to build the dependencies using `yarn install`
 
-## Running the script
+# Building
 
-You can run the script using `node index.js`
\ No newline at end of file
+`yarn build`
+
+# Running
+
+You can run the script using `node built/main.js`
```

Comment:
I think we can  / should update this to `yarn start` now that we have that in our yarn.

Reply:
Done!!